src/Reflection/ReflectionProperty.php

```diff
@@ -197,12 +194,18 @@ public function isInitialized(?object $object = null): bool
             $this->getValue($object);
 
             return true;
-        } catch (Throwable $e) {
-            if ($e instanceof Error && strpos($e->getMessage(), 'must not be accessed before initialization') !== false) {
+
+        // @phpstan-ignore-next-line
+        } catch (Error $e) {
+            if (str_contains($e->getMessage(), 'must not be accessed before initialization')) {
                 return false;
             }
 
+            // @codeCoverageIgnoreStart
+            // @infection-ignore-all
+            // Untestable code
```

Comment:
This path is potentially testable with a class that does `unset($this->prop)`, then declares `__get()` and throws there

Reply:
Nice 👍